scaffolds/java/README.md

```diff
@@ -1,35 +1,47 @@
 # Java Scaffold for Code Retreat
 
-## Option 1: use docker wrapper scripts
+## Option 1: use Docker wrapper scripts
 
-See `scripts/` folder.
+This scaffold comes with pre-configured Docker environment.
 
-## Install Java and Gradle
+### Run the tests
+May need to pull the image (600MB) the 1st time.
+```
+./scripts/test
+```
+Or, using [TCR/BTCR](https://github.com/euranova/code_retreat#tcr):
+```
+./../../scripts/[tcr|btcr]
+```
+
+### Run the app
 
-There are several options. The easiest way is to work with a docker image ready
-with all the tools you need.
+```
+./gradle run
+```
 
-The only downside is that you need to run gradle through Docker. To make it easy, \
-we wrote a script `gradle` wrapping the `docker` command and acting as gradle.
+## Option 2: Install Java and Gradle 4
+TCR/BTCR scripts are not available because they use Docker.
```

Comment:
```suggestion
With this options, TCR/BTCR scripts will not be directly available because they use `./gradle` 
(which calls Docker). If you wish to use TCR without docker, you can change either `./gradle` 
or the scripts in `./scripts/` so that it calls gradle directly without wrapping in a docker.
```

Reply:
will do